examples/apps/generic_client/generic_client.py

```diff
@@ -166,7 +166,7 @@ def Main(args=None):
                 " *****************")
 
     global direct_jrpc
-    direct_jrpc = DirectJsonRpcApiConnector(config_file=None, config=config)
+    direct_jrpc = AvalonDirectClient(config_file=None, config=config)
```

Comment:
Opened an issue to implement Generic client using Avalon SDK APIs for worker registry, work order etc insated of using facade class -> https://github.com/hyperledger/avalon/issues/207